Assets/_Project/Scripts/SpectrobeCard.cs

```diff
@@ -10,16 +11,27 @@ namespace SpectrobesTCG
 
     public class SpectrobeCard : Card
     {
-        public CardManager.SpectrobeForm form = CardManager.SpectrobeForm.adult;
-        public CardManager.SpectrobeProperty property = CardManager.SpectrobeProperty.aurora;
-        public int attack;
-        public int hp;
-
-
+        public SpectrobeData spectrobeData;
+        public TMP_Text spectrobeName;
+        public TMP_Text hpText;
+        public TMP_Text attackText;
+        public Image artwork;
+        public Image frontArt;
```

Comment:
Just as with the ChildCard, this doesn't need to be included because the art is already attached to the prefab and all instances of this prefab will use the same sprite for that particular image. Will wil probably have a separate heirachy of images for the evolved side/face that we will enable the card evolves. This is the Adult/Evolved card forms will be two sides of the same card.

Reply:
I have made changes so that adult spectrobes use the adult card front and when they evolve it changes to the Evolved card front, it works with the Resources.Load function too